lighthouse-core/config/default-config.js

```diff
@@ -369,42 +369,42 @@ const defaultConfig = {
-        {id: 'definition-list', weight: 1, group: 'a11y-well-structured'},
-        {id: 'dlitem', weight: 1, group: 'a11y-well-structured'},
-        {id: 'document-title', weight: 2, group: 'a11y-describe-contents'},
-        {id: 'duplicate-id', weight: 5, group: 'a11y-well-structured'},
-        {id: 'frame-title', weight: 5, group: 'a11y-describe-contents'},
+        {id: 'definition-list', weight: 1, group: 'a11y-tables-lists'},
+        {id: 'dlitem', weight: 1, group: 'a11y-tables-lists'},
+        {id: 'document-title', weight: 2, group: 'a11y-names-labels'},
+        {id: 'duplicate-id', weight: 5, group: 'a11y-best-practices'},
+        {id: 'frame-title', weight: 5, group: 'a11y-names-labels'},
         {id: 'html-has-lang', weight: 4, group: 'a11y-language'},
         {id: 'html-lang-valid', weight: 1, group: 'a11y-language'},
-        {id: 'image-alt', weight: 8, group: 'a11y-correct-attributes'},
-        {id: 'input-image-alt', weight: 1, group: 'a11y-correct-attributes'},
-        {id: 'label', weight: 10, group: 'a11y-describe-contents'},
-        {id: 'layout-table', weight: 1, group: 'a11y-describe-contents'},
-        {id: 'link-name', weight: 9, group: 'a11y-element-names'},
-        {id: 'list', weight: 5, group: 'a11y-well-structured'},
-        {id: 'listitem', weight: 4, group: 'a11y-well-structured'},
-        {id: 'meta-refresh', weight: 1, group: 'a11y-meta'},
-        {id: 'meta-viewport', weight: 3, group: 'a11y-meta'},
-        {id: 'object-alt', weight: 4, group: 'a11y-describe-contents'},
-        {id: 'tabindex', weight: 4, group: 'a11y-correct-attributes'},
-        {id: 'td-headers-attr', weight: 1, group: 'a11y-correct-attributes'},
-        {id: 'th-has-data-cells', weight: 1, group: 'a11y-correct-attributes'},
+        {id: 'image-alt', weight: 8, group: 'a11y-names-labels'},
+        {id: 'input-image-alt', weight: 1, group: 'a11y-names-labels'},
+        {id: 'label', weight: 10, group: 'a11y-names-labels'},
+        {id: 'layout-table', weight: 1, group: 'a11y-tables-lists'},
+        {id: 'link-name', weight: 9, group: 'a11y-names-labels'},
+        {id: 'list', weight: 5, group: 'a11y-tables-lists'},
+        {id: 'listitem', weight: 4, group: 'a11y-tables-lists'},
+        {id: 'meta-refresh', weight: 1, group: 'a11y-best-practices'},
+        {id: 'meta-viewport', weight: 3, group: 'a11y-best-practices'},
+        {id: 'object-alt', weight: 4, group: 'a11y-names-labels'},
+        {id: 'tabindex', weight: 4, group: 'a11y-navigation'},
+        {id: 'td-headers-attr', weight: 1, group: 'a11y-tables-lists'},
+        {id: 'th-has-data-cells', weight: 1, group: 'a11y-tables-lists'},
         {id: 'valid-lang', weight: 1, group: 'a11y-language'},
-        {id: 'video-caption', weight: 4, group: 'a11y-describe-contents'},
-        {id: 'video-description', weight: 3, group: 'a11y-describe-contents'},
+        {id: 'audio-caption', weight: 4, group: 'a11y-audio-video'},
```

Comment:
is the new sorting scheme alphabetical unless it's audio-caption? :)

Reply:
I think I did that one first while I was still figuring out if the audits were sorted by id or group. Fixed